processing/src/main/java/org/apache/druid/query/aggregation/any/DoubleAnyAggregatorFactory.java

```diff
@@ -112,6 +115,18 @@ public BufferAggregator factorizeBuffered(ColumnSelectorFactory metricFactory)
     }
   }
 
+  @Override
+  public VectorAggregator factorizeVector(VectorColumnSelectorFactory selectorFactory)
+  {
+    return new DoubleAnyVectorAggregator(selectorFactory.makeValueSelector(fieldName));
+  }
+
+  @Override
+  public boolean canVectorize(ColumnInspector columnInspector)
+  {
+    return true;
```

Comment:
Should this check the columnType before offering a vectorized implementation? Similar question for other factories